agent/tool-scripts/postprocess/openshift-metrics-postprocess

```diff
@@ -7,8 +7,8 @@ output_file = sys.argv[2]
 in_csv = []
 metrics=[]
 
-with open(input_file, 'r') as f:
-    reader = csv.reader(f)
+with open(input_file, 'r') as file:
+    reader = csv.reader(null_bytes.replace('\0', '') for null_bytes in file)
```

Comment:
I am also curious about what this script is doing.  It appears to be reading `$master-stdout.txt` as a `.csv` file and transforming it into another `.csv` file.  Why not just emit the original `.csv` file and be done with it?